src/Microsoft.DocAsCode.Build.Engine/DocumentBuildContext.cs

```diff
@@ -193,6 +201,26 @@ private List<string> ResolveByExternalReferencePackages(List<string> uidList, Co
             return list;
         }
 
+        private XRefSpec FindByRequest(string uid)
+        {   
+            
+            XRefSpec xf = null;
+            string requestUrl = XRefServerUrls[0];
+            string url = requestUrl + "/" + uid + "/";
+            var client = new HttpClient();
+            if (Uri.TryCreate(url, UriKind.Absolute, out Uri uri))
+            {
+                var response = client.GetAsync(uri).Result;
```

Comment:
or time out

---
In reply to: [126059057](https://github.com/dotnet/docfx/pull/1837#discussion_r126059057) [](ancestors = 126059057)